### PR TITLE
Release Auth AAD 1.2.0

### PR DESCRIPTION
**Pull request by srvrguy**
Update for the 1.2.0 release of the Auth AAD plugin.

### CODE REVIEW

File: authaad.properties [hunk 1]
```diff
@@ -2,12 +2,18 @@ name=Azure Active Directory Authentication for SonarQube
...
 defaults.mavenArtifactId=sonar-auth-aad-plugin
 
+1.2.0.description=Group sync enhancements including transitive members. Code cleanup.
+1.2.0.sqVersions=[7.0, LATEST]
+1.2.0.date=2020-02-05
+1.2.0.changelogUrl=https://github.com/hkamel/sonar-auth-aad/releases/tag/1.2.0
+1.2.0.downloadUrl=https://github.com/hkamel/sonar-auth-aad/releases/download/1.2.0/sonar-auth-aad-plugin-1.2.0.jar
+
 1.1.0.description=Support group sync, national cloud, fix display name for null values, support SQ 7.2 and newer. If upgrading from earlier release, make sure to read the upgrade notes.
 1.1.0.sqVersions=[7.0, LATEST]
```
Comment (ganncamp) [inline authaad.properties]: Only one version can be compatible with `LATEST` at a time. The best practice is to set the end range of the old version to the current SQ version when you release a new plugin version

File: authaad.properties [hunk 2]
```diff
@@ -2,12 +2,18 @@ name=Azure Active Directory Authentication for SonarQube
...
-publicVersions=1.1.0
-archivedVersions=1.0
+publicVersions=1.2.0
+archivedVersions=1.0,1.1.0
 
 defaults.mavenGroupId=org.almrangers.auth.aad
 defaults.mavenArtifactId=sonar-auth-aad-plugin
 
+1.2.0.description=Group sync enhancements including transitive members. Code cleanup.
+1.2.0.sqVersions=[7.0, LATEST]
```
Comment (ganncamp) [inline authaad.properties]: SQ versions <7.9 have been archived. Please set the beginning of this range to 7.9.

File: authaad.properties (hunk 1 above)
Comment (srvrguy) [reply authaad.properties]: Made the requested changes in a new commit in the PR.

File: authaad.properties (hunk 2 above)
Comment (srvrguy) [reply authaad.properties]: Made the requested changes in a new commit in the PR.

**Pull request merged**